README.md

```diff
@@ -42,7 +42,346 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+### Conventions
+
+  * Templates are searched under `Lotus::Mailer.configuration.root`, set this value according to your app structure (eg. `"app/templates"`).
+  * A mailer will look for a template with a file name that is composed by its full class name (eg. `"articles/index"`).
+  * A template must have two concatenated extensions: one for the format and one for the engine (eg. `".html.erb"`).
+  * The framework must be loaded before rendering the first time: `Lotus::Mailer.load!`.
+
+### Mailers
+
+A simple mailer looks like this:
+
+```ruby
+require lotus/mailer
+
+class InvoiceMailer
+  include Lotus::Mailer
+end
+```
+
+A mailer with `to` and `from` addresses and mailer delivery:
+
+```ruby
+require lotus/mailer
+
+class InvoiceMailer
+  include Lotus::Mailer
+  
+  from 'noreply@sender.com'
+  to 'noreply@recipient.com'
+  subject 'Welcome'
+end
+
+CustomDelivery = {
+  :address              => "smtp.gmail.com",
+  :port                 => 587,
+  :domain               => "localhost:8000",
+  :user_name            => "noreply@sender.com",
+  :password             => "password",
+  :authentication       => "plain",
+  :enable_starttls_auto => true
+}
+
+Lotus::Mailer.configuration do
+  delivery_method CustomDelivery
+end
+
+Lotus::Mailer.load!
+InvoiceMailer.deliver
+```
+
+### Locals
+
+The set of objects passed in the `deliver` call are called `locals` and are used to render templates that use objects with the same symbol.
+
+```ruby
+require lotus/mailer
+
+class User
+  def initialize(name, username)
+    @name = name
+    @username = username
+  end
+
+  attr_reader :name, :username
+end
+
+luca = User.new('Luca', 'jodosha')
+InvoiceMailer.deliver(user:luca)
+```
+
+The corresponding `erb` file:
+
+```erb
+Hello <%= user.name %>!
+```
+
+### Scope
+
+All the methods defined in the mailer are accessible from the template:
```

Comment:
@jodosha what do you think about explicitly exposure like those of `Lotus::View`? I kinda like this idea of exposing data through `expose` DSL